tests/test_component/test_amgr.py

```diff
@@ -22,6 +23,8 @@
 
 host     =     os.environ.get('RMQ_HOSTNAME', 'localhost')
 port     = int(os.environ.get('RMQ_PORT',      5672))
+username = os.environ.get('RMQ_USERNAME', 'guest')
+password = os.environ.get('RMQ_PASSWORD', 'guest')
 user     = 'guest'
```

Comment:
Can `user` and `passwd` guest be eliminated now that we use no defaults?

Reply:
Yes, 
1c7362a removed the default values